llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -2556,6 +2576,8 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
   }
   case Intrinsic::spv_step:
     return selectStep(ResVReg, ResType, I);
+  case Intrinsic::spv_radians:
+    return selectRadians(ResVReg, ResType, I);
```

Comment:
first rebase your branch. then do this
```suggestion
    return selectExtInst(ResVReg, ResType, I, CL::radians, GL::Radians);
```